docs/api/proposed_endpoints.md

```diff
@@ -356,3 +356,163 @@ Status: 200 OK
     "posted_time": null
 }
 ```
+
+# Proposals
+
+## Create Proposal
+
+```
+POST /api/proposal
+```
+
+**Parameters**
+
+Name               | Data type     | Description
+-------------------|---------------|---------------------
+id                 | UUID          | Unique ID for the proposal
+name               | text          | Name of proposal
+speaker            | text          | Speaker for the talk
+type               | choices       | Type of proposal like talk, dev sprint, workshop
```

Comment:
Rather than `type`, let's use `kind` here. So, that we avoid using reserved Python keywords in models and elsewhere.